backend/src/test/kotlin/org/loculus/backend/controller/submission/SubmissionConvenienceClient.kt

```diff
@@ -516,4 +521,19 @@ class SubmissionConvenienceClient(
                 .contentAsString
         return objectMapper.readValue(content)
     }
+
+    /**
+     * Upload a file (content) to a presigned write URL (S3).
+     */
+    fun uploadContent(presignedWriteUrl: String, content: String) {
```

Comment:
Why not just name it `uploadFile` - i.e., why introduce the term `content`? I think that we're not using this term otherwise and I found it a bit confusing.

Reply:
Hmm, not sure why the name was chosen. But I've changed it now! https://github.com/loculus-project/loculus/pull/4227/commits/9d72abd25922018fdf547ddc58385b2d0ce2cf31